src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -0,0 +1,122 @@
+package nearsoft.academy.bigdata.recommendation;
+
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+import org.apache.mahout.cf.taste.common.TasteException;
+import org.apache.mahout.cf.taste.impl.model.file.FileDataModel;
+import org.apache.mahout.cf.taste.impl.neighborhood.ThresholdUserNeighborhood;
+import org.apache.mahout.cf.taste.impl.recommender.GenericUserBasedRecommender;
+import org.apache.mahout.cf.taste.impl.similarity.PearsonCorrelationSimilarity;
+import org.apache.mahout.cf.taste.model.DataModel;
+import org.apache.mahout.cf.taste.neighborhood.UserNeighborhood;
+import org.apache.mahout.cf.taste.recommender.RecommendedItem;
+import org.apache.mahout.cf.taste.recommender.UserBasedRecommender;
+import org.apache.mahout.cf.taste.similarity.UserSimilarity;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+
+public class MovieRecommender {
+
+    private DataModel model;
+    private UserSimilarity similarity;
+    private UserNeighborhood neighborhood;
+    private UserBasedRecommender recommender;
+    private List<RecommendedItem> recommendations;
+
+// se usa biMap porque tiene el metodo inverse()
+// que cambia llaves por valores. Util a la hora de buscar
+// uno de los elementos por su valor
+
+    private BiMap<String, String> users;
+    private BiMap<String, String> products;
+    private int totalReviews = 0;
```

Comment:
You may not initialize the value of this variable here, because the default value for an int as a member class is 0 :)